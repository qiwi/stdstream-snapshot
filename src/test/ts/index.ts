import index from '../../main/ts'

describe('', () => {
  it('', () => {
    expect(!!index).toBeTruthy()
  })
})
