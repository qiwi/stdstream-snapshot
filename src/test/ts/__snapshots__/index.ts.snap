// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stdstream-snapshot #generateSnapshot returns a new snapshot 1`] = `
{
  "err": {
    "code": 1,
    "killed": false,
    "signal": null,
  },
  "stderr": "",
  "stdout": "/rules/some-rules.ts
    2:7   error  'name' is assigned a value but never used                                 no-unused-vars
    3:7   error  'name' is already defined                                                 no-redeclare",
}
`;
